src/Tizen.Multimedia.Camera/Camera/CameraDeviceManager.cs

```diff
@@ -113,6 +113,36 @@ public IEnumerable<CameraDeviceInformation> SupportedDevices
             }
         }
 
+        /// <summary>
+        /// Gets the product id for given <param name="device"/>.
```

Comment:
please write "ID" in the text, same for line 132,
lines 126 and 141 can stay unchanged

Reply:
Fixed